### PR TITLE
Unit test for ask tool stacks [#96]:

### LINKED ISSUE #96: Unit test for ask tool stucks
This test has two issues:

First, a simple, wrong directory is used for helpers. This leads to this error:

```text
 ./tests/bats/bin/bats ./tests/ask.bats
ask.bats
 ✗ ask contains -e: should error code without correct parameter
   (from function `setup' in test file tests/ask.bats, line 26)
     `setup() {' failed
   bats_load_safe: Could not find '/home/igor_lysak/hub-extensions/tests/test_helper/bats-support/load'[.bash]
 ✗ ask env: should print messages if parameters need to set for different component discriminators
   (from function `setup' in test file tests/ask.bats, line 26)
     `setup() {' failed
   bats_load_safe: Could not find '/home/igor_lysak/hub-extensions/tests/test_helper/bats-support/load'[.bash]
 ✗ ask env: should success if env exist
   (from function `setup' in test file tests/ask.bats, line 26)
     `setup() {' failed
   bats_load_safe: Could not find '/home/igor_lysak/hub-extensions/tests/test_helper/bats-support/load'[.bash]
 ✗ ask env: should print messages if parameters already set for different component discriminators
   (from function `setup' in test file tests/ask.bats, line 26)
     `setup() {' failed
   bats_load_safe: Could not find '/home/igor_lysak/hub-extensions/tests/test_helper/bats-support/load'[.bash]
 ✗ ask env: should print messages if parameter need to set without different component discriminators
   (from function `setup' in test file tests/ask.bats, line 26)
     `setup() {' failed
   bats_load_safe: Could not find '/home/igor_lysak/hub-extensions/tests/test_helper/bats-support/load'[.bash]
 ✗ ask env: should print messages if parameter already set without different component discriminators
 ✗ teardown_file failed                                                                                                                                                                                                                          `setup() {' failed
   bats_load_safe: Could not find '/home/igor_lysak/hub-extensions/tests/test_helper/bats-support/load'[.bash]
   (from function `teardown_file' in test file tests/ask.bats, line 90)
     `rm .env .ask.env' failed
   removed '.ask.env'
   removed 'ask.params-component.yaml'
   rm: cannot remove '.env': No such file or directory
   rm: cannot remove '.ask.env': No such file or directory
```

Second. After fixing the name of helpers directory test was stuck on the second test case.

```text
 ./tests/bats/bin/bats ./tests/ask.bats
ask.bats
 ✓ ask contains -e: should error code without correct parameter
   ask env: should print messages if parameters need to set for different component discriminators  
```

### PR DESCRIPTION
**Pull request by NellyHargeliya**
- add simulates the user entering for tests
- corrected "helpers" folders for load
- removed creation and deletion of redundant file ".ask.env"

### CODE REVIEW

File: tests/ask.bats [hunk 1]
```diff
@@ -41,51 +43,52 @@ setup() {
   assert_failure
 }
 
+# This test simulates user input for ARGO_HOST and KUBEFLOW_HOST parameters
 @test "ask env: should print messages if parameters need to set for different component discriminators" {
-  run  ask env "ARGO_HOST" -m "parameter ingress.hosts test1" +empty -d ".ask.env" -d ".env" -ask-env
+  run ask env "ARGO_HOST" -m "parameter ingress.hosts test1" +empty -d ".env" -ask-env <<<"test_value_for_ARGO_HOST"
```
Comment (TheRealSpaceShip) [inline tests/ask.bats]: Instead of `".env"` better to use `DOTENV_FILE` env var. Same in other test cases.

File: tests/ask.bats [hunk 2]
```diff
@@ -41,51 +43,52 @@ setup() {
...
+  run ask env "BATS_TEST_HOST" -m "parameter test.hosts test3" +empty -d ".env" -ask-env
   assert_success
   assert_output --partial "* Setting parameter test.hosts test3, BATS_TEST_HOST"
   assert_output --partial "BATS_TEST_HOST already set"
 }
 
 teardown_file() {
-  rm -fv "$DOTENV_FILE" ask.*.yaml
-  rm .env .ask.env
+  rm -fv ask.*.yaml
```
Comment (TheRealSpaceShip) [inline tests/ask.bats]: I think it's better to remove yaml file like `rm "$HUB_FILES"`

File: tests/ask.bats [hunk 3]
```diff
@@ -41,51 +43,52 @@ setup() {
...
 
 @test "ask env: should print messages if parameter already set without different component discriminators" {
-  run  ask env "BATS_TEST_HOST" -m "parameter test.hosts test3" +empty -d ".ask.env" -d ".env" -ask-env
+  run ask env "BATS_TEST_HOST" -m "parameter test.hosts test3" +empty -d ".env" -ask-env
   assert_success
   assert_output --partial "* Setting parameter test.hosts test3, BATS_TEST_HOST"
   assert_output --partial "BATS_TEST_HOST already set"
 }
 
 teardown_file() {
```
Comment (TheRealSpaceShip) [inline tests/ask.bats:90]: Also, need to remove of `.hub` directory.

File: tests/ask.bats (hunk 1 above)
Comment (NellyHargeliya) [reply tests/ask.bats]: done

File: tests/ask.bats (hunk 2 above)
Comment (NellyHargeliya) [reply tests/ask.bats]: done

File: tests/ask.bats (hunk 3 above)
Comment (NellyHargeliya) [reply tests/ask.bats:90]: done

**Comment by akranga** [comment]
@NellyHargeliya please check my comment [here](https://github.com/epam/hub-extensions/issues/96#issuecomment-1665116572)

**Review by TheRealSpaceShip — APPROVED** [review]
Looks good to me

**Pull request merged**